### PR TITLE
feat(telecom): add no meetings message and enable next step

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #UXCT-558
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Add no meetings message and enable next step button

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js [hunk 1]
```diff
@@ -182,6 +182,12 @@ export default class TelecomPackMigrationConfirmCtrl {
   }
 
   getMeeting() {
+    if (
+      !this.process.selectedOffer.meetingSlots ||
```
Comment (antonymarion) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js:186]: use ?.

File: packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js [hunk 2]
```diff
@@ -182,7 +182,10 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
     });
 
     // Set meeting
-    if (migrationProcess.selectedOffer.meetingSlots) {
+    if (
+      migrationProcess.selectedOffer.meetingSlots &&
```
Comment (antonymarion) [inline packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js:186]: same

**Review by antonymarion — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.controller.js [hunk 3]
```diff
@@ -322,7 +325,10 @@ export default class MoveResumeCtrl {
         });
       }
 
-      if (this.offer.selected.meetingSlots) {
+      if (
+        this.offer.selected.meetingSlots &&
+        this.offer.selected.meetingSlots.slot
+      ) {
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.controller.js:331]:
```suggestion
      if (
        this.offer.selected.meetingSlots?.slot
      ) {
```

**Review by ghyenne — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

**Comment by anooparveti** [comment]
Rebasing with release branch to resolve conflicts.

**Pull request merged**